README.md

```diff
@@ -12,127 +12,105 @@ description: "This sample shows how to use Azure KeyVault from App Service with
-AzureServiceTokenProvider will use **Azure CLI** or **Active Directory Integrated Authentication** to authenticate to Azure AD to get a token. That token will be used to fetch the secret from Azure Key Vault. 
+Windows
 
-Azure CLI will work if the following conditions are met:
- 1. You have [Azure CLI 2.0](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli?view=azure-cli-latest) installed. Version 2.0.12 supports the get-access-token option used by AzureServiceTokenProvider. If you have an earlier version, please upgrade. 
- 2. You are logged into Azure CLI. You can login using **az login** command.
- 
-Azure Active Directory Authentication will only work if the following conditions are met:
- 1. Your on-premise active directory is synced with Azure AD. 
- 2. You are running this code on a domain joined machine.   
+```cmd
+setx KEY_VAULT_URI "<YourKeyVaultUrl>"
+```
+
+### Deploy the Web App to Azure
+Use any of the methods outlined on [Deploy your app to Azure App Service] to publish the Web App to Azure.
 
-Since your developer account has access to the Key Vault, you should see the secret on the web page. Principal Used will show type "User" and your user account. 
+Step 1: Set environment variable in app service.
 
-You can also use a service principal to run the application on your local development machine. See the section "Running the application using a service principal" later in the tutorial on how to do this. 
+*  Search for your app service in **Search Resources dialog box**
+*  Select **Setting** > **Configuration** > **New application setting**
+*  Set the name to **KEY_VAULT_URI** and value with your **key vault url**  
 
-## Step 6: Deploy the Web App to Azure
-Use any of the methods outlined on [Deploy your app to Azure App Service](https://docs.microsoft.com/en-us/azure/app-service-web/web-sites-deploy) to publish the Web App to Azure. 
 After you deploy it, browse to the web app. You should see the secret on the web page, and this time the Principal Used will show "App", since it ran under the context of the App Service. 
 The AppId of the MSI will be displayed. 
 
 ## Summary
 The web app was successfully able to get a secret at runtime from Azure Key Vault using your developer account during development, and using MSI when deployed to Azure, without any code change between local development environment and Azure. 
 As a result, you did not have to explicitly handle a service principal credential to authenticate to Azure AD to get a token to call Key Vault. You do not have to worry about renewing the service principal credential either, since MSI takes care of that.  
 
-## Running the application using a service principal in local development environment
```

Comment:
Please add a section here with the same header, but how to use AzureCliCredential instead.

Reply:
updated